### PR TITLE
Updated numpy docstrings 

### PR DESCRIPTION
**Pull request by paarongiroux**
May have skipped over or miss-understood certain things

### CODE REVIEW

File: ale/base/base.py [hunk 1]
```diff
@@ -194,27 +235,51 @@ def sensor_model_version(self):
 
     @abc.abstractproperty
     def focal2pixel_lines(self):
+        """
+        Returns
+        -------
+        : int
```
Comment (jessemapel) [inline ale/base/base.py]: This is a 3 element array of affine transformation coefficients. The coefficients are constant, x coefficient, then y coefficient.  For example, if you have a focal plane coordinate (x,y) and this is F, then the pixel line coordinate is F[0] + x * F[1] + y * F[2].

File: ale/base/base.py [hunk 2]
```diff
@@ -194,27 +235,51 @@ def sensor_model_version(self):
...
+          Focal plane to detector lines
+        """
         pass
 
     @abc.abstractproperty
     def focal2pixel_samples(self):
+        """
+        Returns
+        -------
+        : int
```
Comment (jessemapel) [inline ale/base/base.py]: Same comments as for focal2pixels_lines

File: ale/base/base.py [hunk 3]
```diff
@@ -194,27 +235,51 @@ def sensor_model_version(self):
...
+          Focal plane to detector samples
+        """
         pass
 
     @abc.abstractproperty
     def pixel2focal_x(self):
+        """
+        Returns
+        -------
+        : int
```
Comment (jessemapel) [inline ale/base/base.py]: This is similar to focal2pixels, except it is 3 coefficients for the transformation from pixels to focal plane coordinates and the coefficients are constant, sample, then line. So, if the pixel coordinate is (sample, line) and the coefficient array F, then the focal plane x coordinate is F[0] + sample * F[1] + line * F[2].

File: ale/base/base.py [hunk 4]
```diff
@@ -194,27 +235,51 @@ def sensor_model_version(self):
...
+          detector to focal plane x
+        """
         pass
 
     @abc.abstractproperty
     def pixel2focal_y(self):
+        """
+        Returns
+        -------
+        : int
```
Comment (jessemapel) [inline ale/base/base.py]: See pixel2focal_x comment

File: ale/base/base.py [hunk 5]
```diff
@@ -194,27 +235,51 @@ def sensor_model_version(self):
...
 
     @abc.abstractproperty
     def ephemeris_start_time(self):
         """
           Returns
         -------
-        : str
-          Start time of the image in UTC YYYY-MM-DDThh:mm:ss[.fff]
+        : double
+          Ephemeris start time of the image
```
Comment (jessemapel) [inline ale/base/base.py]: We should be more precise for these, they are "The start(stop) time of the image in ephemeris seconds past the J2000 epoch."

File: ale/base/base.py [hunk 6]
```diff
@@ -223,7 +288,7 @@ def ephemeris_stop_time(self):
         """
           Returns
         -------
-        : str
-          Stop time of the image in UTC YYYY-MM-DDThh:mm:ss[.fff]
+        : double
+          Ephemeris stop time of the image
```
Comment (jessemapel) [inline ale/base/base.py]: See ephemeris start time comment

File: ale/base/data_naif.py [hunk 7]
```diff
@@ -235,29 +388,85 @@ def sensor_orientation(self):
 
     @property
     def ephemeris_start_time(self):
+        """
+        Returns the starting ephemeris time of the image. Expects spacecraft_id to
+        be defined. This must be the integer Naif Id code for the spacecraft. Expects
+        spacecraft_clock_start_count to be defined. This must be a floating point number
```
Comment (jessemapel) [inline ale/base/data_naif.py]: The spacecraft_clock_start_count is actually a string. It's just one of those weird things about SPICE.

File: ale/base/data_naif.py [hunk 8]
```diff
@@ -235,29 +388,85 @@ def sensor_orientation(self):
...
+        """
         return spice.scs2e(self.spacecraft_id, self.spacecraft_clock_start_count)
 
     @property
     def ephemeris_stop_time(self):
+        """
+        Returns the ephemeris stop time of the image. Expects spacecraft_id to
+        be defined. This must be the integer Naif Id code for the spacecraft.
+        Expects spacecraft_clock_stop_count to be defined. This must be a floating point number
+        containing the stop clock count of the spacecraft
```
Comment (jessemapel) [inline ale/base/data_naif.py]: See ephemeris_start_time comment

File: ale/base/label_isis.py [hunk 9]
```diff
@@ -27,15 +29,33 @@ def instrument_id(self):
 
     @property
     def platform_name(self):
+        """
+        Returns the name of the platform containing the sensor
```
Comment (jessemapel) [inline ale/base/label_isis.py]: I would mention that this is usually the spacecraft name.

File: ale/base/label_pds3.py [hunk 10]
```diff
@@ -85,6 +111,8 @@ def spacecraft_name(self):
     @property
     def utc_start_time(self):
         """
+        Returns the start time of the image
```
Comment (jessemapel) [inline ale/base/label_pds3.py]: Mention that it is a UTC string.

File: ale/base/label_pds3.py [hunk 11]
```diff
@@ -249,6 +302,12 @@ def exposure_duration(self):
     # Consider expanding this to handle units
     @property
     def line_exposure_duration(self):
+        """
```
Comment (jessemapel) [inline ale/base/label_pds3.py]: This is the time between the exposures for subsequent lines.

File: ale/drivers/mro_drivers.py [hunk 12]
```diff
@@ -188,12 +236,30 @@ def spacecraft_name(self):
...
+        Returns
+        -------
+        : int
+          Starting detector sample for the image
+        """
         return self.label.get('SAMPLE_FIRST_PIXEL', 0)
 
     @property
     def sensor_model_version(self):
+        """
```
Comment (jessemapel) [inline ale/drivers/mro_drivers.py]: This is the ISIS sensor model version.

Comment (paarongiroux) [reply ale/drivers/mro_drivers.py]: all of the above changes have been made

**Pull request merged**